lib/px/utils/pxapi.lua

```diff
@@ -68,6 +69,7 @@ function M.load(config_file)
 
```

Comment:
line 68 should be replaced with the new enrichment mechanism

Reply:
fixed